framework/routing/__init__.py

```diff
@@ -51,7 +51,6 @@
     http.FOUND,
 ]
 
-
```

Comment:
Not sure this newline should have been removed per [perp8](https://www.python.org/dev/peps/pep-0008/#blank-lines).